src/core/morph.js

```diff
@@ -0,0 +1,54 @@
+import "idiomorph"
+import { nextAnimationFrame } from "../util"
+
+export class Morph {
+  static render(currentElement, newElement, morphStyle) {
+    const morph = new this(currentElement, newElement)
+
+    morph.render(morphStyle)
+  }
+
+  constructor(currentElement, newElement) {
+    this.currentElement = currentElement
+    this.newElement = newElement
+  }
+
+  render(morphStyle = "outerHTML") {
+    window.Idiomorph.morph(this.currentElement, this.newElement, {
+      ignoreActiveValue: true,
```

Comment:
The [ignoreActiveValue:](https://github.com/bigskysoftware/idiomorph/tree/4231990153895d058848b3153efc7e1d2ed2727a#options) option was introduced in `0.0.9`, and feels important enough to include as a default for the sake of focus preservation and to minimize disruptions to the element with focus.

Progressing this branch forward hinges on https://github.com/bigskysoftware/idiomorph/pull/12 being merged. Without it, the package needs to depend on `window.Idiomorph`.